### PR TITLE
ci(client): auto assign reviewers github action

### PR DESCRIPTION
**Pull request by vuhuucuong**
See #29

### CODE REVIEW

**Comment by zlatanpham** [comment]
@vuhuucuong can you update so the person creating a PR will be assigned to a assignee of that PR? 

image
As you will be automatically assigned to the PR

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+addReviewers: true
```
Comment (zlatanpham) [inline .github/auto_assign.yml]:
```
addAssignees: author
```

File: .github/auto_assign.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  - vuhuucuong
+  - zlatanpham
+  - 4nmt
+  - danndz
+  - NguyenDiem
+  - putrann
+  - trongnguyen2706
+  - vdhieu
+skipKeywords:
+  - wip
```
Comment (zlatanpham) [inline .github/auto_assign.yml]: How exactly does it work? Can we skip the job if the PR is in `draft`?

Comment (vuhuucuong) [reply .github/auto_assign.yml]: If the PR's title includes `wip` keyword (case insensitive),  then this action will be skipped
Draft PRs are auto skipped as I saw here https://github.com/kentaro-m/auto-assign-action/blob/master/src/handler.ts#L47-L52

File: .github/auto_assign.yml (hunk 1 above)
Comment (vuhuucuong) [reply .github/auto_assign.yml]: added

**Pull request merged**